From: Anthony G. Basile <basile@opensource.dyc.edu>

We don't want to allow GRSEC's Makefile to change permissions on
paths in the filesystem.

--- a/grsecurity/Makefile	2010-05-21 06:52:24.000000000 -0400
+++ b/grsecurity/Makefile	2010-05-21 06:54:54.000000000 -0400
@@ -22,8 +22,8 @@
 ifdef CONFIG_GRKERNSEC_HIDESYM
 extra-y := grsec_hidesym.o
 $(obj)/grsec_hidesym.o:
-	@-chmod -f 500 /boot
-	@-chmod -f 500 /lib/modules
-	@-chmod -f 700 .
-	@echo '  grsec: protected kernel image paths'
+	# @-chmod -f 500 /boot
+	# @-chmod -f 500 /lib/modules
+	# @-chmod -f 700 .
+	# @echo '  grsec: protected kernel image paths'
 endif
